galley/pkg/config/analysis/analyzers/auth/servicerolebindings.go

```diff
@@ -52,6 +52,11 @@ func (s *ServiceRoleBindingAnalyzer) analyzeRoleBinding(r *resource.Entry, ctx a
 	srb := r.Item.(*v1alpha1.ServiceRoleBinding)
 	ns, _ := r.Metadata.Name.InterpretAsNamespaceAndName()
 
+	// If no servicerole is defined at all, just skip. The field is required, but that should be enforced elsewhere.
+	if srb.RoleRef == nil {
```

Comment:
Can srb itself ever be nil?

Reply:
Coming from non-testing paths, I don't believe so.